tests/test_mks_client_bsn_hr.py

```diff
@@ -3,86 +3,96 @@
 from unittest import TestCase
 from unittest.mock import patch
 
-from mks.service import mks_client_bsn_hr
+from mks.service import mks_client_hr
 
 FIXTURE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'fixtures')
 BSN_RESPONSE_PATH = os.path.join(FIXTURE_PATH, "hr_bsn_response.xml")
 BSN_NO_RESPONSE_PATH = os.path.join(FIXTURE_PATH, "hr_bsn_no_hr_response.xml")
 
+BSN_HR_RESPONSE = {
```

Comment:
Ik zet dit altijd in de return van een functie, dan krijg je namelijk altijd een nieuw dict ipv een referentie naar deze ene.
mocht je wat data willen aanpassen in een test, heb je geen side effects van andere tests.